hack/common.sh

```diff
@@ -27,6 +26,7 @@ function compile() {
   # e.g. IMAGE_PREFIX=username/kpack-
   IMAGE_PREFIX=${IMAGE_PREFIX:-"${registry}/"}
 
+  kpack_version=${kpack_version:="dev"}
```

Comment:
The kpack version is set to 

dev-${COMMIT_SHA}

on push to main and `vX.Y.Z` when a matching version tag is published.